README.md

```diff
@@ -42,14 +42,27 @@ Read about [contentful content types](https://github.com/contentful/contentful-m
 |`instanceOf`  | `null`              |                     | ignored by default         |
 |prop name starts with `_`  | `null` |                     | ignored by default (ex. `_id`) |
 
-### Custom proptype validators:
-We offer a few custom proptype validators to use in your code base, in order to reference specific content models (see an [example here](https://github.com/wework/monarch/blob/master/example/components/CrossSellGroup.jsx)).
-
-|Validator          | Contentful type     | Notes                       |
-|-------------------|---------------------|-----------------------------|
-|`childType()`      | `Link`              | to reference                |
-|`customPropType()` | `Link`              | to reference                |
-|`validateAll()`    | `Array` of `Link`s  | to reference                |
+### Reference validators:
+We recommend using AirBnb's [`airbnb-prop-types`](https://github.com/airbnb/prop-types) for custom proptype validations, we support the following:
+
+|Validator             | Contentful type     | Notes                       |
+|----------------------|---------------------|-----------------------------|
+|`childrenOfType()`    | `Link`              | to reference                |
+|`childrenOfType()`    | `Array` of `Link`s  | needs `@array` jsdoc flag   |
+|`componentWithName()` | `Link`              | to reference                |
```

Comment:
Should we support both of these?

Reply:
Which do you mean exactly?

Reply:
I'm wondering if we should allow both `childrenOfType` and `componentWithName`. They seem to do more-or-less the same thing. If that's the case, is there a benefit in supporting both of them or should we try to limit the API?

Reply:
One argument for using `componentWithName` over `childrenOfType` is that consumers would be free to redefine the functionality of the component as long as its component name remains the same.

Reply:
Oh I see. Supporting both just allows the user to use whichever they like without us making them choose one over the other.

Reply:
Looking at the [airbnb docs](https://github.com/airbnb/prop-types) it seems like they are similar, but _slightly_ different. I'm guessing there was a good reason for implementing both of them so might as well allow both.

Reply:
Unclear of the benefit of having both but we'll probably get a better idea as we use them more. Let's move forward with both for the time being.